html-forms/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <div>
```

Comment:
You're missing a `form` element here. There should be 1 `form` element that wraps all of this content